<ui version="4.0" >
 <class>XMLServer_IFClass</class>
 <widget class="QWidget" name="XMLServer_IFClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>XMLServer_IF</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>591</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Connection</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>SvrAddr</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>BasePort</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>WaitTime</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="txtSvr" >
        <property name="text" >
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spnPort" >
        <property name="minimum" >
         <number>8666</number>
        </property>
        <property name="maximum" >
         <number>9999999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="spnWaitTime" >
        <property name="minimum" >
         <number>5000</number>
        </property>
        <property name="maximum" >
         <number>99999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="textEdit" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>100</y>
       <width>561</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>320</y>
       <width>331</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="btnConnect" >
        <property name="text" >
         <string>&amp;Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnDisConnect" >
        <property name="text" >
         <string>&amp;DisConnect</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStatus" >
        <property name="text" >
         <string>&amp;Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>Select</string>
    </attribute>
    <widget class="QTextEdit" name="txtSelect" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>10</y>
       <width>481</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>SelectSQL</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtResult" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>120</y>
       <width>481</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Result</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>340</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="btnSelect" >
        <property name="text" >
         <string>&amp;Select</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnFirst" >
        <property name="text" >
         <string>&amp;First</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext" >
        <property name="text" >
         <string>&amp;Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3" >
    <attribute name="title" >
     <string>INSERT</string>
    </attribute>
    <widget class="QTextEdit" name="txtAddSql" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>10</y>
       <width>481</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>AddSql</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>AppendSql</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtAppendSql" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>170</y>
       <width>481</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>340</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAppend" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>340</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Append</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4" >
    <attribute name="title" >
     <string>UPDATE</string>
    </attribute>
    <widget class="QTextEdit" name="txtUpdSql" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>10</y>
       <width>491</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>57</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>ExecSQL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpdate" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>250</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete" >
     <property name="geometry" >
      <rect>
       <x>110</x>
       <y>250</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>410</y>
     <width>591</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QLabel" name="statusBar" >
      <property name="minimumSize" >
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth" >
       <number>2</number>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit" >
      <property name="text" >
       <string>&amp;Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
